webapp/src/components/post_menu_actions/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -40,33 +40,20 @@ export default class AttachCommentToIssuePostMenuAction extends PureComponent {
     }
 
     render() {
-        if (this.props.isSystemMessage) {
+        const conn = this.props.connected || {};
```

Comment:
Why is this an object? Looking at the reducer for `connected` it looks like we save a bool. But the mapStateToProps doesn't seem to use that, it seems to just grab any state. Can we fix this strange behavior?

Reply:
Hey @crspeller.  I add an isConnected selector in the latest commit.  Please review to see if that fix is what you had in mind.  Thanks